src/v/kafka/server/handlers/find_coordinator.h

```diff
@@ -14,6 +14,6 @@
 
 namespace kafka {
 
-using find_coordinator_handler = handler<find_coordinator_api, 0, 2>;
+using find_coordinator_handler = handler<find_coordinator_api, 0, 3>;
```

Comment:
why is this being changed? we don't yet support the flexible protocol

Reply:
it's an oversight, keeping the old version